### DISCUSSION TITLE
Setting up a second computer

### DISCUSSION THREAD

**Discussion by magthe** [chep/copilot-chat.el#46] [Q&A] [answered]
I have copilot-chat working fine one of my machines, but I can't get it to work on my second machine. I'm using the same github account on both machines.

The login reports no errors, but every time I try to send a message all I get is

```
copilot-chat--ask: Wrong type argument: number-or-marker-p, nil
```

I'm not sure where to start tracking down the reason for this, but I did find out that my `copilot-chat--instance` looks very different on the machine that doesn't work. It looks something like this:

```
#s(copilot-chat t "ghu_xxxx"
                ((error_details
                  (url . "https://github.com/github-copilot/signup?editor={EDITOR}")
                  (message . "No access to GitHub Copilot found. You are currently logged in as myuser.")
                  (title . "Sign up for GitHub Copilot")
                  (notification_id . "no_copilot_access"))
                 (message . "Resource not accessible by integration"))
                "aaaaaaaa-bbbb-cccc-dddd-..."
                "0123456789abcdef..."
                nil nil)
```

Am I just missing something obvious, or is it not supported?

**Comment by chep** [comment, marked as answer]
It works on  different machines for me. I think there was an error during login process. Could you delete `~/.cache/copilot-chat` and `~/.config/copilot-chat` and try again ?

**Reply by magthe** [reply]
Ah, there's a cache too... I didn't find that one.

Yes, deleting both the config and the cache allowed me successfully log in and now it's working on my second machine as well.
